portfolio/src/main/webapp/commentScript.js

```diff
@@ -12,39 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random suggestion by many to the page.
- */
-function getComment() {
-  console.log('Fetching a comment.');
+/*
+* Purpose: recieves HTTP promise response from CommentsServlet.java, places authors and comments into appropriate html container 
+*/
+
+function getComment(){
+    const responsePromise = fetch('/commentServlet');
+    // when server request complete, pass response into handleResponse
+    responsePromise.then(handleResponse);
+}
 
-  // The fetch() function returns a Promise because the request is asynchronous.
-  const responsePromise = fetch('/commentServlet');
+function handleResponse(response){
+    // receives Java Object (ArrayList reponse)
+    const commentListPromise = response.json();
+    commentListPromise.then(addCommentsToDom);
+}
 
-  // When the request is complete, pass the response into handleResponse().
-  responsePromise.then(handleResponse);
+/** Adds user and reflection to html page **/
+function addCommentsToDom(commentList){
+    const commentContainer = document.getElementById('comments-container');
+    commentContainer.innerHTML = '';
+    for (let i = 0; i < commentList.length; i++){
+        commentContainer.appendChild(
+            createComment(commentList[i].user, commentList[i].reflection));
+    }
 }
 
 /**
- * Handles response by converting it to text and passing the result to
- * addQuoteToDom().
- */
-function handleResponse(response) {
-  console.log('Handling the response.');
-
-  // response.text() returns a Promise, because the response is a stream of
-  // content and not a simple variable.
-  const textPromise = response.text();
-
-  // When the response is converted to text, pass the result into the
-  // addQuoteToDom() function.
-  textPromise.then(addCommentToDom);
+ * The above code is organized to show each individual step, but we can use an
```

Comment:
This comment is unnecessary. If you want to use arrow notation, do it. It's shorter but harder to read/debug. Generally we prefer ease of reading/debugging because you only write software once, but that's personal preference.